api/src/main/java/gov/cms/ab2d/api/util/HealthCheck.java

```diff
@@ -30,8 +30,7 @@ public boolean healthy() {
                 DatabaseAvailable.isDbAvailable(dataSource) &&
                 // EFS Mount can be written to
                 FileSystemCheck.canWriteFile(efsMount, true) &&
-                // Can write to home directory
-                FileSystemCheck.canWriteFile(".", false) &&
```

Comment:
Why was this deleted?

Reply:
So it breaks the CI build in a really inconvenient and difficult to handle way. It also doesn't make sense because you really shouldn't be writing to the working directory of the api container `usr/src/ab2d-api` where the JAR that the container runs is.

Reply:
Is the deploy Dockerfile different from `ab2d/api/Dockerfile`? This is what I get when running `ls` on the `/usr/src/ab2d-api` directory in the container. If this is what is in production then we are publishing everything from the build as part of the docker image.

![image](https://user-images.githubusercontent.com/19293463/92038426-f86bfe80-ed38-11ea-8001-58eab417298d.png)